code/game/turfs/open/floor/fancy_floor.dm

```diff
@@ -126,8 +126,9 @@
 	transform = MAP_SWITCH(TRANSLATE_MATRIX(-9, -9), matrix())
 
 /turf/open/floor/grass/no_border
-	smoothing_groups = list(SMOOTH_GROUP_TURF_OPEN)
-	canSmoothWith = list()
+	smoothing_groups = list(SMOOTH_GROUP_TURF_OPEN, SMOOTH_GROUP_OPEN_FLOOR)
+	canSmoothWith = list(SMOOTH_GROUP_TURF_OPEN, SMOOTH_GROUP_OPEN_FLOOR)
```

Comment:
if you are disabling smoothing, you can just set these lists to null otherwise the smoother will get confused and connect to these, or is that intended?

Reply:
Intended, otherwise it can look kinda weird with the skirt overlapping